.github/pull_request_template.md

```diff
@@ -3,5 +3,8 @@
 -
 -
 
-## security considerations
+:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/BRANCH_NAME)
```

Comment:
is the intent here that a user should be manually replacing `BRANCH_NAME` with the name of their branch?

Reply:
Correct. 

Also how its done in other parts of TTS: https://github.com/18F/18f.gsa.gov/pull/3151

If you can figure out a way to inject the branch name as a variable into the PULL_REQUEST template, you'll make a lot of friends here.

Reply:
gotcha.